packages/argo-checkout/src/extension-points/api/shared.ts

```diff
@@ -1 +1,70 @@
 export type ValueOrPromise<T> = T extends PromiseLike<any> ? T : T | Promise<T>;
+
```

Comment:
I've moved types used for the standard api to this shared file in order to keep the api file clean with only `InputData`, `Storage` and `StandardApi` and avoid circular references.